### DISCUSSION TITLE
Beware of talking about this programm on r/feedthebeast

### DISCUSSION THREAD

**Discussion by bastetfurry** [SKCraft/Launcher#522] [General]
Hey :)

Just got banned on r/feedthebeast for speaking about this program, so beware mentioning it there.
They think this is a cracked client or something.

king regards,
Alexandra

![Bildschirmfoto vom 2023-11-13 14-08-34](https://github.com/SKCraft/Launcher/assets/3686292/ce5e2225-a94a-4220-8711-fd2d95dfd566)

**Comment by hedgehog1029** [comment]
when referring to the launcher, I recommend calling it "SKCLauncher" or the full "SKCraft Launcher" to avoid confusion. "SKLauncher" is indeed a separate client that allows cracked accounts.
